docs/spec/broker.md

```diff
@@ -54,10 +54,10 @@ with that particular Broker, while the `TriggerSpec.Delivery`, if configured,
 fully overrides `BrokerSpec.Delivery` for that particular Trigger, hence:
 
 - When `BrokerSpec.Delivery` and `TriggerSpec.Delivery` are both not configured,
-  no delivery spec MUST be used.
+  no delivery spec SHOULD be used.
```

Comment:
wait, why are we changing all of these? I can see changing the TriggerSpec.Delivery but why all of them?

Reply:
That's fine, atm `BrokerSpec.Delivery` it's not compliant in all brokers (like in mtbroker)

Reply:
@slinkydeveloper @zhongduo I don't see why we needed to change all these to support backwards compatibility.

Reply:
I am not familiar with all the other brokers, but slinkydeveloper seems to make the point that this is not supported by all the brokers too. I am good with keeping the origin too, though we use MUST rarely.

Reply:
@vaikas do we want this spec to "map to the reality", or the "reality map to the spec"?